octodns/provider/route53.py

```diff
@@ -29,19 +31,84 @@ def _octal_replace(s):
 class _Route53Record(object):
 
     @classmethod
-    def new(self, provider, record, creating):
+    def _new_dynamic(cls, provider, record, hosted_zone_id, creating):
         ret = set()
-        if getattr(record, 'geo', False):
-            ret.add(_Route53GeoDefault(provider, record, creating))
-            for ident, geo in record.geo.items():
-                ret.add(_Route53GeoRecord(provider, record, ident, geo,
-                                          creating))
-        else:
-            ret.add(_Route53Record(provider, record, creating))
+
+        # HostedZoneId wants just the last bit, but the place we're getting
+        # this from looks like /hostedzone/Z424CArX3BB224
+        hosted_zone_id = hosted_zone_id.split('/', 2)[-1]
+
+        # Create the default pool
+        fqdn = record.fqdn
+        ret.add(_Route53Record(provider, record, creating,
+                               '_octodns-default-pool.{}'.format(fqdn)))
+
+        # Pools
+        for pool_name, pool in record.dynamic.pools.items():
+
+            # Create the primary
+            ret.add(_Route53DynamicPool(provider, hosted_zone_id, record,
+                                        pool_name, creating))
+
+            # Create the fallback
+            fallback = pool.data.get('fallback', False)
+            if fallback:
+                # We have an explicitly configured fallback
+                ret.add(_Route53DynamicPool(provider, hosted_zone_id, record,
+                                            pool_name, creating,
+                                            target_name=fallback))
```

Comment:
We have an explicit fallback configured. This SECONDARY will alias to that fallback.